dev-python/python-gammu/python-gammu-2.11.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python bindings for Gammu"
+HOMEPAGE="https://wammu.eu/python-gammu/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples test"
+
+RDEPEND=">=app-mobilephone/gammu-1.34.0"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	test? (
+		$(python_gen_impl_dep sqlite)
+		app-mobilephone/gammu[dbi]
+	)"
+
+DOCS=( AUTHORS NEWS.rst README.rst )
+
+python_test() {
+	esetup.py test
+}
+
+python_install_all() {
+	use examples && local EXAMPLES=( examples/. )
```

Comment:
Please test it with `USE=examples` enabled.

Reply:
tested. USE=test for dev-python/python-gammu require USE=dbi for gammu. with gammu[dbi] work perfect

Reply:
I also don't believe that it worked